packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
+
+**Best practices**
```

Comment:
The "best practice" section mainly describes the behavior of the DatePicker. For me, a "best practice" section should mostly tell me in which case I should use a component and why. WDYT?

Reply:
Both works to me, at least for a first iteration. If you have any suggestions to add anything to this section, do not hesitate to let me know :)